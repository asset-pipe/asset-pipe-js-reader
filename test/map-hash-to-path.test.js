'use strict';

const mapHashToPath = require('../lib/map-hash-to-path');

test('returns map of feed ids -> feed file paths', () => {
    const feed = require('./mock/feed.c.json');

    expect(Array.from(mapHashToPath(feed))).toMatchSnapshot();
});
