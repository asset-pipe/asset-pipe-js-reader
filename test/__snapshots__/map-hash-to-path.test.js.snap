// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`returns map of feed ids -> feed file paths 1`] = `
Array [
  Array [
    "3c4ca5444d554f82cb15c2437ed545b2ea6f6a6e98476ec372b21b6cf7d8085a",
    "/Users/ricwalke/src/asset-pipe/asset-pipe-js-reader/test/mock/podletA/entrypoint.js",
  ],
  Array [
    "94def8251ca7023be6c5aeda886a4055d4d47fd62238228def5888398bbdf318",
    "/Users/ricwalke/src/asset-pipe/asset-pipe-js-reader/test/mock/podletA/lib/index.js",
  ],
]
`;
